client/securedrop_client/gui/conversation/export/export_wizard_page.py

```diff
@@ -381,14 +383,14 @@ def __init__(self, export: Export) -> None:
     @pyqtSlot(object)
     def on_status_received(self, status: ExportStatus) -> None:
         self.status = status
-        self.update_content(status)
+        self.update_content(status, should_show_hint=False)
```

Comment:
This page has its own message ("Remember to be careful working with files outside the workstation"), which we display instead of the hint ("Export successful").  I could also change the hint to be our reminder text, and remove the boolean parameter to `update_content` entirely.